### DISCUSSION TITLE
disableInteraction not working React Native

### DISCUSSION THREAD

**Discussion by MDmubarak786** [elrumordelaluz/reactour#442] [Q&A]
please help me how to do this !!!
doSteps is it function ?

`import { TextPlaceholder, doSteps } from '../utils'
const demoId = 'disable-interaction'
const steps = doSteps(demoId)

;<TourProvider
  steps={steps}
  onClickHighlighted={e => {
    e.stopPropagation()
    console.log('No interaction')
  }}
  disableInteraction
>
  <TextPlaceholder demoId={demoId} />
</TourProvider>`

**Comment by elrumordelaluz** [comment]
Hi @MDmubarak786,
I [pinned](https://github.com/elrumordelaluz/reactour/issues) [this issue](https://github.com/elrumordelaluz/reactour/issues/69) since I am not sure the _Tour_ could work on RN.
However, mind create a reduced reproduction in a sandbox, in order to play around?
Thanks,

**Reply by MDmubarak786** [reply]
i asking how to do in your site not saying how to do can you tell me how disable the interaction while overlay come

**Reply by MDmubarak786** [reply]
i was working for this since last week please help me to make disable then interaction

**Comment by elrumordelaluz** [comment]
[disableInteraction](https://github.com/elrumordelaluz/reactour/tree/master/packages/tour#disableinteraction-boolean) prop avoid the interaction with the highlighted element, as in [this example](https://reactour.vercel.app/#disable-interaction) where you can't select the text highlighted in the step.

Is this behaviour you are looking for?

Mind playing a little with a codesandbox, like [this one](https://codesandbox.io/s/tour-with-data-fetching-dv2q0?file=/src/index.js) and come back to tell what is exactly which isn't working for you?

**Reply by MDmubarak786** [reply]
yes i am looking for this only but not working in my project

**Reply by MDmubarak786** [reply]
from your codeSandbox Code case while overlay coming in "Get Text" button right i need to disable that interaction if overlay coming on any container it needs to be disable the interaction
